src/WebJobs.Script.WebHost/Diagnostics/SystemLogger.cs

```diff
@@ -181,8 +180,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                 formattedMessage = Sanitizer.Sanitize(formattedMessage);
                 innerExceptionMessage = innerExceptionMessage ?? string.Empty;
             }
-
-            _eventGenerator.LogFunctionTraceEvent(logLevel, subscriptionId, appName, functionName, eventName, source, details, summary, innerExceptionType, innerExceptionMessage, invocationId, _hostInstanceId, activityId, runtimeSiteName, slotName, DateTime.UtcNow);
+            _eventGenerator.LogFunctionTraceEvent(logLevel, subscriptionId, appName, functionName, eventName, source, details, formattedMessage, innerExceptionType, innerExceptionMessage, invocationId, _hostInstanceId, activityId, runtimeSiteName, slotName, DateTime.UtcNow);
```

Comment:
Null coalesce `formattedMessage` to empty string.

Reply:
There is already a check for [`IsNullEmpty`](https://github.com/Azure/azure-functions-host/blob/351083550777eadf9fe793823ea03d7be1c33e00/src/WebJobs.Script.WebHost/Diagnostics/SystemLogger.cs#L122)`